uat/custom-components/client-java-paho/src/main/java/com/aws/greengrass/testing/mqtt5/client/paho/MqttConnectionImpl.java

```diff
@@ -357,9 +369,11 @@ public void messageArrived(String topic, MqttMessage mqttMessage) {
 
             String contentType = null;
             Boolean payloadFormatIndicator = null;
+            Long messageExpiryInterval = null;
```

Comment:
messageExpiryInterval does not used, actually it just logged.

Nor sets to GRPCClient.MqttReceivedMessage nor send to control by gRPC.